### PR TITLE
ARS-613: content for pages up to valuationMethod

### PR DESCRIPTION
**Pull request by C-Thomp**
[ARS-613](https://jira.tools.tax.service.gov.uk/browse/ARS-613)

### CODE REVIEW

File: app/models/ValuationMethod.scala [hunk 1]
```diff
@@ -50,9 +50,11 @@ object ValuationMethod extends Enumerable.Implicits {
 
   def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
     case (value, index) =>
+      val x = messages(s"valuationMethod.${value.toString}")
+
       RadioItem(
         content = HtmlContent(
-          Html(messages(s"valuationMethod.${value.toString}"))
+          Html(s"<b>${messages(s"valuationMethod.${value.toString}")}</b>")
         ),
```
Comment (RawToast) [inline app/models/ValuationMethod.scala:58]: I think you can remove the `x` line here

**Review by RawToast — APPROVED** [review]
🍪

File: app/models/ValuationMethod.scala (hunk 1 above)
Comment (C-Thomp) [reply app/models/ValuationMethod.scala]: Done

**Review by RawToast — DISMISSED** [review]

File: app/views/ImportingGoodsView.scala.html [hunk 2]
```diff
@@ -73,10 +75,8 @@
 @layout(pageTitle = titleNoForm(messages("importingGoods.title"))) {
 
     @caption(messages("importingGoods.caption"))
-
-    <h1 class="govuk-heading-xl">@messages("importingGoods.heading")</h1>
```
Comment (RawToast) [inline app/views/ImportingGoodsView.scala.html:77]: 🎉

**Review by RawToast — APPROVED** [review]
🥇

**Pull request merged**